kafka/channel/pkg/dispatcher/dispatcher_test.go

```diff
@@ -1,19 +1,31 @@
 package dispatcher
 
 import (
+/*
```

Comment:
this seems like formatting is all messed up? Should be above package start of the file.